.github/workflows/check-upstream.main.kts

```diff
@@ -36,7 +37,8 @@ workflow(
                 git clone --branch master --single-branch https://bitbucket.org/snakeyaml/snakeyaml-engine.git
                 wget https://raw.githubusercontent.com/krzema12/snakeyaml-engine-kmp/${'$'}{{ github.ref }}/upstream-commit.txt
                 cd snakeyaml-engine
-                git log --oneline $(cat ../upstream-commit.txt)..master | wc -l > ../$numberOfCommitsFileName
+                git log --oneline ${'$'}(cat ../upstream-commit.txt)..master > ../$logDiffBetweenRepos
```

Comment:
I initially wanted to somehow build a link to BitBucket (e.g. https://bitbucket.org/snakeyaml/snakeyaml-engine/branches/compare/master..ef7ebe9c06e963e13f4ab465f300a0dd6f0940c9), but I failed to come up with an elegant approach how to build the link dynamically. One idea was to have an `<a href="...">` element in the SVG, but it would require us to edit the SVG that shields.io returns. So for now I thought such simple text file will suffice.

Reply:
If I am not mistaken, the shields.io provides API to insert a link into the badge ([link parameter](https://shields.io/badges/static-badge)). E.g.:

https://img.shields.io/badge/To%20upstream-42-blue?link=https%3A%2F%2Fbitbucket.org%2Fsnakeyaml%2Fsnakeyaml-engine%2Fbranches%2Fcompare%2Fmaster..ef7ebe9c06e963e13f4ab465f300a0dd6f0940c9

Unfortunately, I am not near the computer right now and cannot check if the link works correctly

Reply:
Nice find! However it doesn't seem to work if the image is in a Markdown doc, probably due to:

> Note that this only works when integrating your badge in an <object> HTML tag, but not an <img> tag or a markup language.

Reply:
Ah...okay. In this case, what do you think about adding that link as the first line to the file with commits diff? It is not ideal but at least the link will be near by

Reply:
Yes, good idea!

Reply:
Done: https://github.com/krzema12/snakeyaml-engine-kmp/pull/264/commits/1a4aa28c17caa39b907cdc5a1622e3e7ecc0434a